pyaedt/generic/spisim.py

```diff
@@ -0,0 +1,208 @@
+
+        cfgCmmd = '-i %s -v CFGFILE="%s"' % (touchstone_file, config_file)
+        if out_file:
+            cfgCmmd += ' -o "%s"' % out_file
+        command = [spisimExe, parameter, cfgCmmd]
+        # Debug('%s %s' % (cmdList[0], ' '.join(arguments)))
+        # try up to three times to be sure
+        out_processing = os.path.join(self.working_directory, generate_unique_name("spsim_out") + ".txt")
+        my_env = os.environ.copy()
+        my_env.update(settings.aedt_environment_variables)
+        if is_linux:  # pragma: no cover
+            command.append("&")
+            with open(out_processing, "w") as outfile:
+                subprocess.Popen(command, env=my_env, stdout=outfile, stderr=outfile).wait()  # nosec
+        else:
+            with open(out_processing, "w") as outfile:
+                subprocess.Popen(" ".join(command), env=my_env, stdout=outfile, stderr=outfile).wait()  # nosec
+        return out_processing
+
+    @pyaedt_function_handler()
+    def _get_output_parameter_from_result(self, out_file, parameter_name):
+        try:
+            with open(out_file, "r") as infile:
+                lines = infile.read()
+                parmDat = lines.split("[ParmDat]:", 1)[1]
+                for keyValu in parmDat.split(","):
+                    dataAry = keyValu.split("=")
+                    if dataAry[0].strip().lower() == parameter_name.lower():
+                        return float(dataAry[1].strip().split()[0])
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+        except IndexError:
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+
+    @pyaedt_function_handler()
+    def compute_erl(
+        self,
+        config_file=None,
+        port_order=None,
+        specify_through_ports=None,
+        bandwidth=None,
+        tdr_duration=None,
+        z_terminations=None,
+        transition_time=None,
+        fixture_delay=None,
+        input_amplitude=None,
+        ber=None,
+        pdf_bin_size=None,
+        signal_loss_factor=None,
+        permitted_reflection=None,
+        reflections_lenght=None,
+        modulation_type=None,
+    ):
+        """Compute effective return loss (erl) using Ansys SPISIM from a S-parameter file.
+
+        Parameters
+        ----------
+        config_file : str, optional
+            Configuration file to use as a reference. The default is ``None``, in
+            which case this parameter is ignored.
+        port_order : str, optional
+            Whether to use "``EvenOdd``" or "``Incremental``" numbering for ``s4p`` files.
+            Ignored if the ports are greater than 4.
+        specify_through_ports : list, optional
+            Input and Output ports on which compute the erl. Those are ordered like ``[inp, inneg, outp, outneg]``.
```

Comment:
```suggestion
            Input and output ports to compute the erl on. Those are ordered like ``[inp, inneg, outp, outneg]``.
```
What is "erl"? should it be ERL or spelled out.

Reply:
It is defined in the definition of the method _Compute effective return loss (erl)_